.gitignore

```diff
@@ -1,5 +1,4 @@
 output.json
-data.yaml
```

Comment:
Is there a reason you removed it? Since each local environment may have a different data.yaml (a file that specifies which rules are activated), it should be ignored.

Reply:
Because I've implemented the metadata in `data.yaml` and we've started utilizing OPA's capability of bundling the data YAMLs.
I want to be able to add and commit `data.yaml` in different places of the source code and bundle them.
For example when we will switch metadata for EKS as well.
If we want to ignore a specific `data.yaml` in specific paths, maybe we should write that explicitly in the `.gitignore`?
What do you think?

Reply:
yeah makes sense, so in that case,  we would want to ignore the `data.yaml` in the root directory, but not in other places, so something like `/data.yaml` will do the trick, right?

Reply:
Yes, I've pushed it right now and also added the output/input JSON as well to be checked under root folder